packages/salesforcedx-vscode-core/src/commands/source/sourceTrackingGetStatusExecutor.ts

```diff
@@ -26,13 +26,15 @@ export class SourceTrackingGetStatusExecutor extends LibraryCommandletExecutor<
     this.options = options;
   }
 
-  public async execute(): Promise<void> {
+  public async execute(): Promise<boolean> {
```

Comment:
Now that the return type is boolean, I feel using a `try catch` and returning false instead of undefined will make more sense to me.